src/minsk.sln

```diff
@@ -2,15 +2,15 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.30011.22
 MinimumVisualStudioVersion = 15.0.26124.0
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Minsk", "Minsk\Minsk.csproj", "{6FE6F7CC-1626-4731-9B68-7236B5A4C94C}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Minsk", "Minsk\Minsk.csproj", "{6FE6F7CC-1626-4731-9B68-7236B5A4C94C}"
```

Comment:
The old project `Guild` is for non-sdk style projects. The new one is for the sdk-style projects. I don't know why `dotnet sln` adds this instead of the new one. Live unit testing sometimes break if the `Guid` is wrong. One time it did work. So, I don't know. Better safe than sorry, I guess.